src/utils/helpers/buildMapper.ts

```diff
@@ -10,17 +10,19 @@ const STATUS_OPTIONS: any = {
 
 const TIMEZONE = Intl.DateTimeFormat().resolvedOptions().timeZone;
 
-const isBuilding = (status: string) => {
-  const BUILD_STATUS = ["BUILDING"];
-  return BUILD_STATUS.includes(status);
-};
-
-const isQueued = (status: string) => {
-  const BUILD_STATUS = ["QUEUED"];
-  return BUILD_STATUS.includes(status);
-};
+const isCompleted = ({
```

Comment:
Maybe this is just a matter of preference but based just on this function's name I'd expect this to return a bool.

One other thing: this is only being called in `buildStatus` below, do we need to break this out into a separate function? You're already checking the value of `status` there, so this would be a bit shorter if you weren't checking it again here.